src/requester.ts

```diff
@@ -0,0 +1,89 @@
+import bsv from 'bsv';
+import { AuthHeader, setSignature } from './authentication';
+import { Logger } from './logger';
+import { ErrorNoAdminKey, ErrorResponse } from './errors';
+
+export class Requester {
```

Comment:
### minor
maybe `HttpClient` or `ApiClient` or leave it if you don't like my propositions 🙂

Reply:
So, `HttpClient` it is 👍